src/data.js

```diff
@@ -5887,6 +5887,17 @@ module.exports = [
       'Musician',
     ],
   },
+  {
+    name: 'Dušan Simić',
+    description: 'Computer Science student at University of Novi Sad, Serbia and open source software enthusiast.',
+    url: 'https://dusansimic.me/about/',
```

Comment:
"url" with value "https://dusansimic.me/about/" fails to match the required pattern: /(use|uses|using|setup|environment|

Reply:
Sorry to be a bother 😅 but is it really required for `uses` page to have one of those paths? I added things I use to my about page so that's why I've added that link.